src/pages/reactivistas/index.tsx

```diff
@@ -1,26 +1,127 @@
 import Layout from '../../components/Layout';
-import { createReactGroup, getPersonByDiscordId } from '../../lib/api';
+import { getAllReactGroups } from '../../lib/api';
 import { useForm } from 'react-hook-form';
 import { ReactGroup } from '../../lib/types';
-import { useState } from 'react';
+import { useState, Fragment, FormEvent } from 'react';
 import Modal from '../../components/Modal';
+import { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { usePreviewSubscription } from '../../lib/sanity';
+import { reactGroupQuery } from '../../lib/querys';
+import Hero from '../../components/Hero';
 
-const ReactGroupPage: React.FC = () => {
+const ReactGroupPage: React.FC<
+  InferGetStaticPropsType<typeof getStaticProps>
+> = ({ data, preview }) => {
   const [isModalOpen, setIsModalOpen] = useState(false);
 
+  const { data: groups } = usePreviewSubscription(reactGroupQuery, {
```

Comment:
I'm curious, what's this `usePreviewSubscription ` hook about?

Reply:
@marbiano that hook is added in the pr #34, only show the previews in realtime, when you change any in the sanity, without reload the page show the changes

Reply:
@Fedeya thanks for explaining!